docs/docs/1.2.0/policies/external-services.md

```diff
@@ -23,8 +23,15 @@ spec:
     kuma.io/protocol: http
   networking:
     address: httpbin.org:443
-    tls:
+    tls: # optional
       enabled: true
+      allowRenegotiation: false
+      caCert: # one of inline, inlineString, secret
+        inline: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0t... # Base64 encoded cert
```

Comment:
Is this actually required to be base64 encoded? If so, why? YAML has enough string quoting types that base64 doesn't need to be required, and the Datasource proto doesn't document any base64 requirement.

Reply:
I'm wondering whether we want to provide a hard-coded value here, or whether a variable indicator and letting the user know they need to provide their own value for the cert would be better.

Docs team tends to lean toward variable use, in which case it would look like:

```suggestion
        inline: {YOUR_CERT_VALUE}... # Your Base64 encoded cert
```